### PR TITLE
Start working on functionality for controls component

### PR DESCRIPTION
**Pull request by mike-duke**
i added the functionality to populate the distance drop-down in the controls and started to get the search working. we're having a very weird problem when the list of trails is searched or filtered. we may need to get some help with this on monday. let me know if you see anything else that needs to be addressed.

### CODE REVIEW

**Comment by mike-duke** [comment]
also added the dropdown data for the difficulty. I'm going to try figuring out why we're having this issue with filtering the cards. Let me know if you see anything that needs to be fixed

File: src/Controls.js [hunk 1]
```diff
@@ -30,20 +41,30 @@ export default class Controls extends Component {
...
-        </select>
-        <select>
-
-        </select>
-        <button onClick={this.getDistance}>Submit</button>
+
+          <select onChange={this.updateDistance} className="distance-select">
+            <option>Filter by distance (in miles)</option>
+            {
+              this.props.trails.sort((trailA, trailB) => {
```
Comment (pvinthemix) [inline src/Controls.js]: Nice work here. The logic makes a lot of sense for this functionality.

**Review by pvinthemix — COMMENTED** [review]
resetApplication method looks solid

**Comment by pvinthemix** [comment]
Do you think we need the search input on the landing screen?

File: src/App.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
+import React, {
```
Comment (thatPamIAm) [inline src/App.js:1]: It's common to see the import of React and Component all on one line

File: src/App.js [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
+  Component
+} from 'react';
+// import logo from './logo.svg';
 import './styles/App.scss';
 import TrailList from './TrailList.js';
-import Header from './Header.js';
-import Search from './Search.js'
+// import Header from './Header.js';
```
Comment (thatPamIAm) [inline src/App.js:7]: Remove commented out code

File: src/App.js [hunk 4]
```diff
@@ -13,109 +15,140 @@ import Controls from './Controls.js';
...
-    .catch(error => console.log(error))  
- 
-  fetch('https://whateverly-datasets.herokuapp.com/api/v1/trails')
-    .then(response => response.json())
-    .then(trailData => {
-      this.setState( {
-        trailData: trailData
+      .catch(error => console.log(error))
+
+    fetch('https://whateverly-datasets.herokuapp.com/api/v1/trails')
```
Comment (thatPamIAm) [inline src/App.js:36]: Is this a leftover `fetch` for trails that you meant to remove after the creation of the `fetchTrails` method below it?

**Pull request merged**